### DISCUSSION TITLE
Move Git status icons to the rightest of the view like `neo-tree`

### DISCUSSION THREAD

**Discussion by vollowx** [nvim-tree/nvim-tree.lua#2348] [Q&A]
**Is this a question?**
No

**Can this functionality be implemented utilising API?**
No

**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
`neo-tree` give users ability to put Git status icons on the rightest of the tree view, while `nvim-tree` can only put them before or after the file name, and the status-column

**Describe alternatives you've considered**
Impl it and add it as a option in `renderer.icons.git_placement = 'end'`

**Additional context**
![image](https://raw.githubusercontent.com/nvim-neo-tree/resources/main/images/Neo-tree-with-right-aligned-symbols.png)

**Comment by alex-courtis** [comment]
`:h nvim-tree.renderer.icons.git_placement` can be set to `"after"`

**Reply by vollowx** [reply]
Yes, but not the rightest of the whole view, and icons of the value `after` can be invisible if the width is not enough while `rightest` can stay at the rightest of the view.

**Reply by alex-courtis** [reply]
Raised #2349 for all 3 icons.

As usual, [Pull Requests](https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md) are most gratefully appreciated.
